whiterabbit/src/main/java/org/ohdsi/whiteRabbit/ResourceType.java

```diff
@@ -0,0 +1,29 @@
+package org.ohdsi.whiteRabbit;
+
+import java.util.ArrayList;
+
+public enum ResourceType {
+    DELIMITED("Delimited text files"),
+    MYSQL("MySQL"),
+    ORACLE("Oracle"),
+    MSSQL("SQL Server"),
+    POSTGRESQL("PostgreSQL"),
+    MSACCESS("MS Access"),
+    PDW("PDW"),
+    REDSHIFT("Redshift"),
+    TERADATA("Teradata");
+
+    public final String label;
+
+    ResourceType(String label) {
+        this.label = label;
+    }
+
+    public static ArrayList<String> getAllLabels(){
+        ArrayList<String> labels = new ArrayList<>();
+        for(ResourceType dt: ResourceType.values()) {
+            labels.add(dt.label);
+        }
+        return labels;
+    }
```

Comment:
Since this is only going to be used as an array, I would suggest instead
```java
ResourceType[] values = values();
String[] labels = new String[values.length];
for (int i = 0; i < values.length; i++) {
     labels[i] = values[i].label;
}
return labels;
```